recipes/pybuilder/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pybuilder" %}
+{% set version = "0.11.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea740e0c4794935a401b4f19d67b34c660d6f0147bb135c6918a5c8c563dade9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py==37]
```

Comment:
I think it is better to use the `py37` selector here.  The `py` variable has historically changed type (from int to str) and so I think of it as less stable.

Reply:
I thought `py37` was discouraged over `py==37`: https://github.com/conda/conda-build/pull/3363/files

Reply:
Fair, mike knows better